plugins/org.jkiss.dbeaver.ui.editors.image/src/org/jkiss/dbeaver/ui/controls/imageview/IImageViewer.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ui.controls.imageview;
+
+import java.io.InputStream;
+
+public interface IImageViewer {
+
+    /**
+     * Loads inputStream and produces image from it
+     *
+     * @param inputStream image input stream
+     * @return is image loaded successfully
+     */
+    boolean loadImage(InputStream inputStream);
+
+    /**
+     * Removes an active image from viewer
+     *
+     * @return is image removed successfully
+     */
+    boolean clearImage();
```

Comment:
This method is not used

Reply:
![image](https://user-images.githubusercontent.com/20579256/201612536-c7130a6f-b512-4af2-b2a0-237de7173c74.png)